src/app/pages/OpenWalletPage/Features/FromPrivateKey/index.tsx

```diff
@@ -54,38 +53,18 @@ export function FromPrivateKey(props: Props) {
             {t('openWallet.privateKey.instruction', 'Enter your private key in Base64 format.')}
           </label>
         </Paragraph>
-        <FormField
-          htmlFor="privateKey"
-          error={privateKeyIsValid === false ? t('openWallet.privateKey.error', 'Invalid private key') : ''}
-          border
-          contentProps={{ border: privateKeyIsValid ? false : 'bottom' }}
-          round="small"
+
+        <PasswordField
+          value={privateKey}
+          onChange={onChange}
+          error={privateKeyIsValid ? false : t('openWallet.privateKey.error', 'Invalid private key')}
+          inputElementId="privatekey"
+          placeholder={t('openWallet.privateKey.enterPrivateKeyHere', 'Enter your private key here')}
+          hideTip={t('openWallet.privateKey.hidePrivateKey', 'Hide private key')}
+          showTip={t('openWallet.privateKey.showPrivateKey', 'Show private key')}
           width="xlarge"
-        >
-          <Box direction="row" align="center">
-            <TextInput
-              id="privatekey"
-              data-testid="privatekey"
-              placeholder={t('openWallet.privateKey.enterPrivateKeyHere', 'Enter your private key here')}
-              value={privateKey}
-              onChange={onChange}
-              type={privateKeyIsVisible ? 'text' : 'password'}
-              plain
-            />
-            <Tip
-              content={
-                privateKeyIsVisible
-                  ? t('openWallet.privateKey.hidePrivateKey', 'Hide private key')
-                  : t('openWallet.privateKey.showPrivateKey', 'Show private key')
-              }
-            >
-              <Button
-                onClick={() => setPrivateKeyIsVisible(!privateKeyIsVisible)}
-                icon={privateKeyIsVisible ? <View /> : <Hide />}
-              />
-            </Tip>
-          </Box>
-        </FormField>
+        ></PasswordField>
```

Comment:
This could be a self closing tab.

Reply:
Changed. I don't really care tho

If it is good practice then we should add react/self-closing-comp